stable/traefik/templates/_helpers.tpl

```diff
@@ -148,6 +148,17 @@ Helper for containerPort (http)
 	{{- end -}}
 {{- end -}}
 
+{{/*
+Helper for RBAC Scope
+If Kubernetes namespace selection is defined and the (one) selected
+namespace is the release namespace Cluster scope is unnecessary.
+*/}}
+{{- define "traefik.rbac.scope" -}}
+	{{- if not (eq (.Values.kubernetes.namespaces | default (list) | toString) (list .Release.Namespace | toString)) -}}
```

Comment:
Because `values.kubernetes` may not be defined, you require a few more if/elses

Reply:
Fixed.